netbox/circuits/tables/circuits.py

```diff
@@ -88,3 +89,26 @@ class Meta(NetBoxTable.Meta):
         default_columns = (
             'pk', 'cid', 'provider', 'type', 'status', 'tenant', 'termination_a', 'termination_z', 'description',
         )
+
+
+class CircuitTerminationTable(NetBoxTable):
+    circuit = tables.Column(
+        verbose_name=_('Circuit'),
+        linkify=True
+    )
+    site = tables.Column(
+        verbose_name=_('Site'),
+        linkify=True
+    )
+    provider_network = tables.Column(
+        verbose_name=_('Provider Network'),
+        linkify=True
+    )
+
+    class Meta(NetBoxTable.Meta):
+        model = CircuitTermination
+        fields = (
+            'pk', 'id', 'circuit', 'term_side', 'site', 'provider_network', 'port_speed', 'upstream_speed',
+            'xconnect_id', 'pp_info', 'description', 'created', 'last_updated', 'actions',
+        )
+        default_columns = ('pk', 'id', 'circuit', 'term_side', 'description')
```

Comment:
We should probably add a column for circuit provider, and include it by default. Otherwise it can be difficult to make sense of the ordering.